ocsptools/ocsp_tools.go

```diff
@@ -111,12 +116,25 @@ func CreateOCSPReqFromCert(certFile string, ocspURL string, reqMethod string, ha
 
 // getOCSPResponse constructs and sends an OCSP request then returns the OCSP response
 func GetOCSPResponse(ocspReq *http.Request) ([]byte, error) {
+	startTime := time.Now()
+
 	httpClient := &http.Client{}
 	httpResp, err := httpClient.Do(ocspReq)
 	if err != nil {
 		return nil, fmt.Errorf("Error sending http request: %w", err)
 	}
```

Comment:
Will this ever timeout? Is there a way to set a timeout or have this automatically fail if this has gone beyond the time limit? It might be complicated, but possibly running in a thread and then killing the thread if it hasn't finished in the time limit. If it is too complicated, it can be saved for another PR.

Reply:
The HTTP Client does allow you to set a timeout, I should probably use that field.